docker/run_docker.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+set -ex;
+
+container_name="unitycatalog"
```

Comment:
The current sh file is already complex, how would you feel about adding a minimal docker-compose file?

Reply:
I think it's a great idea, leave it with me and I'll come back with some changes tonight and tomorrow.
Thanks @Fokko :star: 
Also I've seen your other comments, I'll address them individually asap.

Reply:
Awesome, let me know if you run into anything 🙌